import sys


def fatal_error(message):
    print('Error: ' + message)
    sys.exit()